packages/broker/src/plugins/metrics/MetricsPlugin.ts

```diff
@@ -13,6 +13,7 @@ export interface MetricsPluginConfig {
     consoleAndPM2IntervalInSeconds: number
     nodeMetrics: {
         storageNode: string
+        firehoseStreamIdHead: string
```

Comment:
Rename to `streamIdPrefix`, as the firehose pattern is the only pattern we'll support? We concatenate this to a `suffix`, so `prefix` could be a good word for the prefix/head of the string.